hugo-extended.yaml

```diff
@@ -1,32 +1,25 @@
 package:
   name: hugo-extended
   version: 0.123.7
-  epoch: 0
-  description: The world's fastest framework for building websites.
+  epoch: 1
+  description: The world's fastest framework for building websites (with extended SASS libraries).
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
+      - hugo
       - libstdc++
 
 environment:
   contents:
     packages:
       - busybox
-      - ca-certificates-bundle
-      - git
+      - wolfi-baselayout
 
 pipeline:
-  - uses: go/install
-    with:
-      package: github.com/gohugoio/hugo
-      version: v${{package.version}}
-      tags: extended
```

Comment:
we are building with  tags: extended , in the `hugo` package we just build the standard versions 

go install -tags extended github.com/gohugoio/hugo@latest

https://github.com/gohugoio/hugo/blob/master/README.md#build-from-source